src/Continuous/normalization.jl

```diff
@@ -382,6 +390,72 @@ function _normalize(ivp::IVP{LCS{N, IdentityMultiple{N}}, Interval{N, IA.Interva
     return IVP(CLCS(ivp.s.A, Universe(1)), ivp.x0)
 end
 
+# ===========================
+# Second order systems
+# ===========================
+
+# we left the n-dimensional second order system to a 2n-dimensional first
+# order system, assuming that the first n variables correspond to "velocities"
+# and the second n variables to "positions" x̃ = [x', x]
+function _second_order_linear_matrix(M::AbstractMatrix{N}, C, K) where {N}
```

Comment:
let's add an option to choose how to order `x̃ = [x', x]`. by default i would use `x̃ = [x, x']`. the new signature can be:

```julia
function _second_order_linear_matrix(M::AbstractMatrix{N}, C, K;
                                                             derivatives_last=true) where {N}
....
end